doc/modules/cross_validation.rst

```diff
@@ -209,6 +209,28 @@ The following sections list utilities to generate indices
 that can be used to generate dataset splits according to different cross
 validation strategies.
 
+.. _iid_cv
+
+Cross-validation iterators for i.i.d. data
+==========================================
+
+Independent Identically Distributed (i.i.d.) data is simply the one where
```

Comment:
The structure of this sentence is weird. May I suggest:

Assuming that some data is Independent Identically Distributed (i.i.d.) is making the assumption that all samples stem from the same generative process and that the generative process is assumed to have no memory of past generated samples.